packages/teleport/src/Kubes/useKubes.ts

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { useState, useEffect } from 'react';
+import useAttempt from 'shared/hooks/useAttemptNext';
+import useTeleport from 'teleport/useTeleport';
+import { Kube } from 'teleport/services/kube';
+import useStickyClusterId from 'teleport/useStickyClusterId';
+
+export default function useKubes() {
+  const ctx = useTeleport();
```

Comment:
lets extract context, if we ever wanted to test hooks, we can manipulate context's data by passing it in as a parameter